plugins/vpp/aclplugin/vppcalls/acl_vppcalls.go

```diff
@@ -36,6 +37,44 @@ const (
 	ICMPv6Proto = 58
 )
 
+// AclMessages is list of used VPP messages for compatibility check
+var AclMessages = []govppapi.Message{
+	&acl_api.ACLAddReplace{},
+	&acl_api.ACLAddReplaceReply{},
+	&acl_api.ACLDel{},
+	&acl_api.ACLDelReply{},
+	&acl_api.MacipACLAdd{},
+	&acl_api.MacipACLAddReply{},
+	&acl_api.MacipACLDel{},
+	&acl_api.MacipACLDelReply{},
+	&acl_api.ACLDump{},
+	&acl_api.ACLDetails{},
+	&acl_api.MacipACLDump{},
+	&acl_api.MacipACLDetails{},
+	&acl_api.ACLInterfaceListDump{},
+	&acl_api.ACLInterfaceListDetails{},
+	&acl_api.ACLInterfaceSetACLList{},
+	&acl_api.ACLInterfaceSetACLListReply{},
+	&acl_api.MacipACLInterfaceAddDel{},
+	&acl_api.MacipACLInterfaceAddDelReply{},
+}
+
+// GetAclPluginVersion returns version of the VPP ACL plugin
+func GetAclPluginVersion(vppChannel *api.Channel, stopwatch *measure.Stopwatch) (uint32, uint32, error) {
+	defer func(t time.Time) {
+		stopwatch.TimeLog(acl_api.ACLPluginGetVersion{}).LogTimeEntry(time.Since(t))
+	}(time.Now())
+
+	req := &acl_api.ACLPluginGetVersion{}
+	reply := &acl_api.ACLPluginGetVersionReply{}
+	// Does not return retval
+	if err := vppChannel.SendRequest(req).ReceiveReply(reply); err != nil {
+		return 0, 0, fmt.Errorf("failed to get VPP ACL plugin version: %v", err)
+	}
+
+	return reply.Minor, reply.Major, nil
```

Comment:
You should either switch these values so they are in the semantic order (major, minor, patch) or at least name the return values so the users can see which version comes first from the function signature.